R/pkg/R/generics.R

```diff
@@ -735,6 +752,8 @@ setGeneric("between", function(x, bounds) { standardGeneric("between") })
 setGeneric("cast", function(x, dataType) { standardGeneric("cast") })
 
 #' @rdname columnfunctions
+#' @param x a Column object.
+#' @param ... additional argument(s).
```

Comment:
Ah I recall. These are generated by code but are exported functions - you see their names match the generics.

For these I think we should put the roxygen block in the R file that has the generator code, follow by NULL like what we have for write.ml in mllib.R